CA_DataUploaderLib/BaseSensorBox.cs

```diff
@@ -390,26 +384,6 @@ private SensorSample[] GetSamples(MCUBoard board)
             return samples;
         }
 
-        private void HandleSaltLeakage(MCUBoard board, SensorSample sensor)
-        {
-            if (sensor.GetType() == typeof(IOconfSaltLeakage))
-            {
-                if (sensor.Value < 3000 && sensor.Value > 0)  // Salt leakage algorithm. 

```

Comment:
@thomasjamatomics any objections to this removal? as far as I know this style of salt leakage detection configuration is not in use anywhere + can be done by combinations of math + alarms and/or introducing something more closely aligned with any hardware + multi node deployments

Reply:
discussed offline, will remove